src/query/storage/index.go

```diff
@@ -64,10 +64,13 @@ func FromIdentTagIteratorToTags(identTags ident.TagIterator) (models.Tags, error
 
 // TagsToIdentTagIterator converts coordinator tags to ident tags
 func TagsToIdentTagIterator(tags models.Tags) ident.TagIterator {
+	//TODO get a checked bytes pool here instead
```

Comment:
nit: Maybe update this to `// TODO get a Tags and TagsIterator from a ident.Pool here instead of creating both